home-manager/home-manager

```
@@ -248,41 +249,34 @@ function doBuild() {
 
     setFlakeAttribute
     if [[ -v FLAKE_CONFIG_URI ]]; then
-        local exitCode=0
         nix build \
             "${PASSTHROUGH_OPTS[@]}" \
              ${DRY_RUN+--dry-run} \
              ${NO_OUT_LINK+--no-link} \
             "$FLAKE_CONFIG_URI.activationPackage" \
-            || exitCode=1
-        return $exitCode
+            || exit $?
```

I'd prefer to bubble the error code onwards, i.e.,
```suggestion
            || return
```